moveit_kinematics/ikfast_kinematics_plugin/README.md

```diff
@@ -0,0 +1,9 @@
+# MoveIt! IKFast
```

Comment:
This file shows as if it was added rather than moved, not sure that it can be easily fixed but it be nice if it kept the history.